src/common/data/config/appConfig.ts

```diff
@@ -3,3 +3,6 @@ export const HF_INFERENCE_API_KEY = process.env.HF_INFERENCE_API_KEY
 export const NEXT_PUBLIC_SUPABASE_ANON_KEY =
   process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY
 export const NEXT_PUBLIC_SUPABASE_URL = process.env.NEXT_PUBLIC_SUPABASE_URL
+export const SUPABASE_SERVICE_ROLE_KEY = process.env.SUPABASE_SERVICE_ROLE_KEY
+
+export const HOME_PAGE = '/candidate/job-tracker'
```

Comment:
Maybe we should rename this variable CANDIDATE_HOME? It so doesn't matter though.